services/recommendations/train/finetune.py

```
@@ -0,0 +1,53 @@
+
+from transformers import AutoTokenizer
+from transformers import AutoModelForSequenceClassification
+from torch.nn.functional import softmax
+
+tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+# Load pre-trained BERT model for sequence classification
+model = AutoModelForSequenceClassification.from_pretrained("bert-base-uncased", num_labels=numNews)
+
+prompt = ""
+
+for idx, newsData in enumerate(Data[:-1]):
```

Ruff (0.8.2)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)